gradle-plugins/src/main/groovy/local.std.java.library-conventions.gradle

```diff
@@ -8,6 +8,7 @@ plugins {
 
     id 'starter.java.deps-open-tracing-common-conventions'
     id 'starter.java.publish-jar-conventions'
+    id 'starter.java.lint-spotless-java-conventions'
```

Comment:
Crux of this PR. Other files were automatically updated due to `./gradlew  spotlessApply`